authui/src/authflowv2/components/divider.css

```diff
@@ -1,7 +1,7 @@
 @layer components {
   :root {
     /* FIXME(davis): Design use opacity on it which should be not necessary */
-    --divider__bg-color: var(--color-grey-200);
+    --divider__bg-color: var(--color-grey-100);
```

Comment:
oh when was it changed..

Reply:
Is the FIXME still relevant?

Reply:
Should not be, let me remove it.

cc @Ngkaokis

Reply:
Removed in ee8c1c0ff99ca232a47710b44f871f4feb6e6e3b